packages-ts/gauntlet-cosmos-contracts/src/commands/contracts/access_controller/deploy.ts

```diff
@@ -0,0 +1,33 @@
+import { AbstractInstruction, instructionToCommand } from '../../abstract/executionWrapper'
```

Comment:
had to add deploy command since it didn't exist before. Previously ops was still able to deploy thanks to this https://github.com/smartcontractkit/chainlink-cosmos/blob/augustus.BCI-1511.more-gauntlet-tests/packages-ts/gauntlet-cosmos-contracts/src/commands/abstract/index.ts#L74 find by @cfal

Reply:
We can't remove the AbstractInstructions or AbstractCommand. They are inherited by every command (in this case they are called abstract in the OOP-sense not because they are meant for deploy/execution/query commands with no gauntlet command implementation but because concrete gauntlet commands use their abstract methods)